static/js/settings_config.ts

```diff
@@ -227,6 +227,29 @@ export const wildcard_mention_policy_values = {
     },
 };
 
+export const create_web_public_stream_policy_values = {
+    by_moderators_only: {
+        order: 1,
+        code: 4,
+        description: $t({defaultMessage: "Admins and moderators"}),
+    },
+    by_admins_only: {
+        order: 2,
+        code: 2,
+        description: $t({defaultMessage: "Admins only"}),
+    },
+    by_owners_only: {
+        order: 3,
+        code: 7,
+        description: $t({defaultMessage: "Owners only"}),
+    },
+    nobody: {
+        order: 4,
+        code: 6,
```

Comment:
Reading this, I think we end up having a bunch of duplication of the `code` values.  

Would it make sense have this be implemented as something that computes this from `common_policy_values` by just taking a subset of the objects?  

I guess we'd need a separate object that's `common_policy_value_possibilities` or something for the purpose. 

No need to do this as part of this PR, but I'm thinking that could be a nice refactor to simplify this file significantly.